packages/strapi-knex/package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "knex": "^0.13.0",
     "lodash": "^4.17.4",
+    "pg": "^7.3.0",
```

Comment:
IMO, if we install `pg` and `sqlite3`, we should also install `mysql`. However, the package dependencies will become heavy. We should remove all the client dependency.